src/apps/loan-list/materials/stackable-material/material-info.tsx

```diff
@@ -37,7 +39,13 @@ const MaterialInfo: FC<MaterialInfoProps> = ({
           </div>
         </div>
         <div className="list-reservation__about">
-          <h3 className="text-header-h4 color-secondary-gray">{title}</h3>
+          <button
+            onClick={() => openDetailsModal()}
```

Comment:
I have now read several perfomance artikels stating that you should avoid anonymous functions because they are evaluated on every render. 
I think it should be possible to write:
```typescript
onClick={openDetailsModal}
```
instead?

Reply:
sure